### ISSUE TITLE
Either change default coin-type back to 118 or make it mandatory

### ISSUE THREAD

**Issue by gjermundgaraba** [cosmos/relayer#1126]
Right now, the default coin-type is set to `-1`, which is a breaking change, and a silent one at that. It took me some time to realize the default had changed.

```go
cmd.Flags().Int32(flagCoinType, -1, "coin type number for HD derivation")
```

So if we don't want it to be 118, then I would argue it should be set as either a mandatory flag, or added to the mandatory list of arguments. Either is fine, but -1 is not a good option.

If we can decide the path forward to this, I am happy to provide a PR.

**Comment by jtieri** [comment]
Thanks for the issue, this was a bug introduced when we added the `coin-type` field to the chain config. The goal was to support automatically building the derivation path so that users did not have to worry about it. There were no checks on if the `coin-type` field was populated though so it would end up using the default value of `0`. 

this should be fixed in #1134 and will be included in the next RC of v2.3.0

**Comment by jonathanpberger** [comment]
Resolved in [v2.3.0rc3](https://github.com/cosmos/relayer/releases/tag/v2.3.0-rc3).
